packages/expo-device/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- [iOS] Added support for iOS 17. ([#39797](https://github.com/expo/expo/pull/39797) by [@chrfalch](https://github.com/chrfalch))
```

Comment:
Is it iPhone 17 instead of iOS 17?

Reply:
Yes it is - thanks for catching this. Fixed here:

https://github.com/expo/expo/pull/39982